arithmetization/src/main/java/net/consensys/linea/zktracer/module/rlp/txrcpt/RlpTxrcpt.java

```diff
@@ -146,9 +145,20 @@ private void phase2(RlpTxrcptColumns traceValue, Boolean status, Trace trace) {
 
   private void phase3(RlpTxrcptColumns traceValue, Long cumulativeGasUsed, Trace trace) {
     final int phase = 3;
-    Preconditions.checkArgument(cumulativeGasUsed != 0, "Cumulative Gas Used can't be 0");
+    //    Preconditions.checkArgument(cumulativeGasUsed != 0, "Cumulative Gas Used can't be 0");
     rlpInt(
-        1, phase, cumulativeGasUsed, false, false, false, true, false, false, 0, traceValue, trace);
+        1,
+        phase,
+        cumulativeGasUsed == 0 ? 123456 : cumulativeGasUsed,
```

Comment:
please add a big TODO to correct it when it'll be possible

Reply:
It's not fixed, I need **YOU** to fix it :p